Assembus/Assets/Scripts/MainScreen/Sidebar/HierarchyView/HierarchyViewController.cs

```diff
@@ -469,5 +492,61 @@ private void DeselectParent(Transform child)
             if (SelectedItems.Contains(parent)) DeselectItem(parent);
             else DeselectParent(parent.transform);
         }
+
+        /// <summary>
+        ///     Scrolls to the targetItem
+        /// </summary>
+        /// <param name="targetItem"></param>
+        public void ScrollToItem(RectTransform targetItem)
+        {
+            scroll.enabled = false;
+            Canvas.ForceUpdateCanvases();
+            scroll.enabled = true;
+
+            // Top border of the actual viewport
+            var topBorder = contentPanel.localPosition.y;
+
+            // Lower border of the actual viewport
+            var lowerBorder = topBorder + 1000;
+
+            // Target item position in the viewport
+            var itemPosition = scrollRect.InverseTransformPoint(contentPanel.position).y -
+                               scrollRect.transform.InverseTransformPoint(targetItem.position).y;
```

Comment:
Why did you use `scrollRect.InverseTransformPoint` once and `scrollRect.transform.InverseTransformPoint` the other time?